api/app/signals/apps/sigmax/views.py

```diff
@@ -29,6 +30,9 @@ def post(self, request, format=None):
         """
         Handle SOAP requests, dispatch on SOAPAction header.
         """
+        if not request.user.has_perm('signals.perform_sigmax_updates'):
```

Comment:
Can we use `signals.sia_signal_change_status` here?

Reply:
The `signals.sia_signal_change_status` Django permission is available to quite a few users. These could then, theoretically, use knowledge of what nuisance complaints were sent to CityControl to close those cases (exactly the behavior we have now).

This is why I added a permission, there was another suggestion of David's was to make the user that acts on behalf of configurable, and check for that user.